endian/endian.go

```diff
@@ -0,0 +1,14 @@
+package endian
```

Comment:
Can you move this over to the `sys` package?

Reply:
To `sys/endian.go` and usage will be `sys.GetEndian()`.

Reply:
done. incorporated changes.

Reply:
Thanks for renaming. Can you please remove the `endian/endian.go` file too.